ws-test/Startup.cs

```diff
@@ -113,29 +107,36 @@ ValueTask SendMessage(ReadOnlyMemory<byte> message)
                 }
             }
 
-            await webSocket.CloseAsync(webSocket.CloseStatus.Value, webSocket.CloseStatusDescription, CancellationToken.None);
```

Comment:
Not sure about this one (it's working but the doc says we have to check for idle clients and close the socket manually)